OrbitLinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -11,6 +11,10 @@
 #include <array>
 #include <cstdint>
 
+#ifndef NDEBUG
+#include "OrbitBase/Logging.h"
```

Comment:
without it compilation fails on L56

Reply:
Do we upset IWYU if we include it regardless of `NDEBUG`?

Reply:
Ah makes sense. Can you also add `// IWYU pragma: keep` behind the include?

Reply:
done.

Reply:
I think it has to go on the same line as the include itself. Ref: https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUPragmas.md

Reply:
done.